### DISCUSSION TITLE
ERR: no space left on device

### DISCUSSION THREAD

**Discussion by grahamreeds** [bluenviron/mediamtx#376] [Questions] [answered]
Just extracted the `v0.16.0_linux_amd64` to a folder and ran it. I get 7 lines of debug telling me that various listeners are opened and then failing with an `ERR: no space left on device` message.

The system is a Core i7 running Centos 7. I have checked df and the disk is fine (37% full).

Any ideas as to why?  Should I just compile it?

Update: Just enabled debug logging in the .yml file but got no more info.
```bash
$ ./rtsp-simple-server 
2021/05/11 10:51:44 I [0/0] rtsp-simple-server v0.16.0
2021/05/11 10:51:44 I [0/0] [RTSP] UDP/RTP listener opened on :8000
2021/05/11 10:51:44 I [0/0] [RTSP] UDP/RTCP listener opened on :8001
2021/05/11 10:51:44 I [0/0] [RTSP] TCP listener opened on :8554
2021/05/11 10:51:44 I [0/0] [RTMP] listener opened on :1935
2021/05/11 10:51:44 I [0/0] [HLS] listener opened on :8888
2021/05/11 10:51:44 I [0/0] ERR: no space left on device
```

**Comment by grahamreeds** [comment, marked as answer]
Rebooting my PC fixed the problem...

**Reply by polizogo67** [reply]
For me sometimes yes some times no ...

**Comment by kant2002** [comment]
Good to know. What can be done to troubleshoot this error? I suspect that's not disk space is an issue. Error is quite confusing. I have 0.17.8 version

**Comment by lanwin** [comment]
I have the same problem but within a docker container. Not sure what todo now.

**Reply by pvbaudin** [reply]
I'm in the same boat, did you end up figuring out a solution?

**Comment by hiennguyen9874** [comment]
Works for me https://askubuntu.com/a/1088794/1355424

**Reply by monsterlyg** [reply]
this work for me.

[a comment by an automated account is not shown]
